ansible/content-servers.yml

```diff
@@ -0,0 +1,121 @@
+- hosts: bananas_fileservers
+  gather_facts: true
+  become: true
+
+  pre_tasks:
+    - name: Update apt.
+      apt:
+        update_cache: true
+        cache_valid_time: 86400  # a day is plenty
+      changed_when: false
+
+    - name: Upgrade & cleanup apt.
+      apt:
+        upgrade: safe
+        autoremove: yes
+        autoclean: yes
+
+    - name: Install apt https support & cron.
+      apt:
+        name:
+          - apt-transport-https
+          - cron  # required for certbot
+        state: present
+
+  roles:
+    - geerlingguy.certbot
+    - geerlingguy.nginx
+
+  tasks:
+    - name: Flush handlers in case any configs have changed.
+      meta: flush_handlers
+
+    # Note that, in effect, this runs against localhost
+    # Doesn't quite test actual external access, but it's much faster than waiting for my slow internet connection
```

Comment:
I doubt your slow connection would be an issue for downloading `healthz` ... :P I would prefer if this can be done via external access ;)